src/css/_buttons.scss

```diff
@@ -21,4 +21,49 @@
     color: var(--color-primary);
     border-color: var(--color-primary);
   }
+
+  &.il-white-blue {
```

Comment:
I wonder if there's a way to name these so that their purpose is clearer. If instead of "blue" or "orange", they had names like "primary-action" or "secondary-action" or "save" or "cancel". My worry is that people won't understand when to use each variant, and they'll fall back to personal preference or, worse, make arbitrary decisions.

In this way, we can also make these buttons smarter, so that a primary action button changes color depending on its context. Then nobody has to remember what color background something might be sitting on, or deal with situations where it might change depending on the screen size.

The clearer we can be about the intention behind each design variation, the more consistency we can expect and the better our chance of preserving good UX.

Reply:
And I realize that the test pages have no context, so we need these very prosaic class names here and in other cases like it. This is mostly a food-for-thought comment.